core/src/iam/signin.rs

```diff
@@ -130,7 +142,7 @@ pub async fn db_access(
 							sess.ip.clone_from(&session.ip);
 							sess.or.clone_from(&session.or);
 							// Compute the value with the params
-							match kvs.evaluate(val, &sess, vars).await {
+							match kvs.evaluate(val.clone(), &sess, vars).await {
```

Comment:
I just checked `Datastore::evaluate` and I see that this method does not require `val`, it could take just a ref to `val`.
I did it at my local branch. @gguillemas  Do you mind if I do a commit with the fixes?

Reply:
Good catch! Please go ahead :+1: